### PR TITLE
Organize constants

### PR DESCRIPTION
**Pull request by JessyCatterwaul**

### CODE REVIEW

**Comment by JessyCatterwaul** [comment]
This is based on the `delete-self` branch, which should be merged into `development` first.

 Is there a better way to make a chain of pull requests based off of each other? I've got a lot of changes and don't want anyone to have to approve them all at once.

**Comment by 0xTim** [comment]
Dependent on #471

**Comment by sammyd** [comment]
@JessyCatterwaul I'm not sure whether it's the best way, but we often make a PR that is to merge into the branch you're building off. That way the changes are more visible. Then, once the other branch is merged, /this/ PR can be updated to be targeted back at development.

It's a bit of a dance, but it makes reviews easier.

**Comment by sammyd** [comment]
FYI—I've done that this with one.

From what I can tell, this PR doesn't _rely_ on the changes in `delete-self` does it? Or indeed Swift 5.3? I think I'd be tempted to rebase this onto `development`, so that it can be reviewed independently, and merged.

Thoughts @0xTim?

File: Emitron/Emitron/UI/Library/LibraryView.swift
Comment (0xTim) [inline Emitron/Emitron/UI/Library/LibraryView.swift:97]: I personally prefer interpolation over concatenation but happy to be persuaded otherwise. Was there a particular reason for this change?

**Comment by 0xTim** [comment]
@sammyd @JessyCatterwaul yep I think rebasing and merging into development directly would be good for this PR

File: Emitron/Emitron/UI/Library/LibraryView.swift
Comment (JessyCatterwaul) [reply Emitron/Emitron/UI/Library/LibraryView.swift:97]: A lot of the benefit of moving the constants into extensions, rather than using a "namespace", is to be able to take advantage of implicit member expressions. e.g. `.tutorials` instead of `Namespace.tutorials`. I haven't been able to reliably get that to work, with string interpolation.

It does seem that `"\(libraryRepository.totalContentNum) \(.tutorials)"` compiles, now! (I'm surprised by that.*) So I'm inclined to make that commit and push. Keep reading and let me know your opinion on whether I should.

---

* …but it won't compile in a playground. 🤷‍♂️ This is the behavior I've come to expect, everywhere.

```swift
extension String {
  static let tutorials = "Tutorials"
}

"\(.tutorials)" // error: cannot infer contextual base in reference to member 'tutorials'
```

---

When it doesn't work, we can fix it* with a concrete overload of `appendInterpolation` (Refer to [the `DefaultStringInterpolation` source](https://github.com/apple/swift/blob/master/stdlib/public/core/StringInterpolation.swift).):

```swift
public extension DefaultStringInterpolation {
  @inlinable mutating func appendInterpolation(_ string: String) {
    string.write(to: &self)
  }
}
```

I didn't bother (and instead went with the built-in alternative that I incorrectly thought we still needed) because we don't have an extensions package going yet. That's on Catie's and my to-do to talk to Sam about in early September.

_* That `@inlinable` can't get parsed, if used directly in a playground._

**Review by sammyd — APPROVED** [review]
Will merge if CI passes

**Pull request merged**
